sos/policies/cos.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import with_statement
```

Comment:
You shouldnt need this on any supported Python runtime.

Reply:
Thanks!